modules/store/src/selector.ts

```diff
@@ -605,9 +605,9 @@ export function createFeatureSelector(
 ): MemoizedSelector<any, any> {
   return createSelector((state: any) => {
     const featureState = state[featureName];
-    if (isDevMode() && featureState === undefined) {
+    if (isDevMode() && featureName !== 'router' && !(featureName in state)) {
```

Comment:
May I ask what the added benefit is of having `featureName !== 'router'`? The ` !(featureName in state))` part already catches the initial state of undefined for the router.

Reply:
It's unneeded.
Good catch, thanks!